### PR TITLE
Feature/select add items

### PR DESCRIPTION
**Pull request by jshjohnson**

### CODE REVIEW

**Comment by jshjohnson** [comment]
If anybody is interested in this feature, please test it and provide feedback 💪🏼

**Comment by dgt41** [comment]
Works fine for single and multiple options:
screen shot 2017-04-17 at 23 06 38

File: tests/spec/choices_spec.js [hunk 1]
```diff
@@ -333,12 +333,17 @@ describe('Choices', () => {
...
       // Key down to second choice
-      this.choices._onKeyDown({
-        target: this.choices.input,
-        keyCode: 40,
-        ctrlKey: false,
-        preventDefault: () => {}
-      });
+      let count = 0;
+      while (count < 2) {
+        console.log('keydown');
```
Comment (dgt41) [inline tests/spec/choices_spec.js]: Reminder to remove this `console.log`

**Comment by dgt41** [comment]
@jshjohnson any updates here?

**Comment by jshjohnson** [comment]
See update [here](https://github.com/jshjohnson/Choices/issues/39) :)

**Pull request merged**